dataflow/src/ops/rewrite.rs

```diff
@@ -0,0 +1,240 @@
+use std::collections::{HashMap, HashSet};
+use prelude::*;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Rewrite {
```

Comment:
I'd like to see some basic `///` documentation for this operator. A basic outline of what its forward processing does using the variable names in `new()` would go a long way.